pkg/build/api/v1beta1/conversion.go

```diff
@@ -4,6 +4,7 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/conversion"
 	newer "github.com/openshift/origin/pkg/build/api"
+	image "github.com/openshift/origin/pkg/image/api"
```

Comment:
I don't like imageapi (as mentioned elsewhere).  If I could redo this, I would make the root package have the API.  However, when it's used in mixed context I'll use imageapi.

----- Original Message -----
> > @@ -4,6 +4,7 @@ import (
> >  	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
> >  	"github.com/GoogleCloudPlatform/kubernetes/pkg/conversion"
> >  	newer "github.com/openshift/origin/pkg/build/api"
> > +	image "github.com/openshift/origin/pkg/image/api"
> 
> we use to call this `imageapi` (we do in many places)
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/679/files#r23208078